scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -70,6 +69,11 @@ workflow GvsCreateFilterSet {
       project_id = project_id
   }
 
+  Int scatter_count = if GetNumSamplesLoaded.num_samples < 100 then 20
```

Comment:
```suggestion
  Int scatter_count = if GetNumSamplesLoaded.num_samples < 100 then 20
                      else if GetNumSamplesLoaded.num_samples < 1000 then 100
                      else if GetNumSamplesLoaded.num_samples < 10000 then 200
                      else if GetNumSamplesLoaded.num_samples < 100000 then 500
                      else 1000```

Reply:
Might be a little cleaner?

Reply:
I went with the formatting that the Winstanley plugin recommended, although I like the way your suggestion looks,.